build/Makefile

```diff
@@ -35,7 +35,7 @@ release_registry = gcr.io/agones-images
 #
 
 # Version defaults to the short hash of the latest commit
-VERSION ?= $(base_version)-$(shell git rev-parse --short HEAD)
```

Comment:
This is good catch! Should this be in a separate PR? Happy to have it in 1, but we usually squash down to a single commit for each PR.